config/templates/metasploit-framework-wrappers/msfwrapper.erb

```diff
@@ -25,8 +25,7 @@ add_path() {
 
 check_db() {
   if [ ! -e $LOCALCONF/database.yml ]; then
-    while true; do
-      read -p "Would you like to use and setup a new database (recommended)? " yn
+    while read -p "Would you like to use and setup a new database (recommended)? " yn; do
```

Comment:
Will do some validation testing. The `loop` here was intended to account for invalid input. This change seems like it would now continued based on any input value meaning the original intended function to ensure the a defined answer is lost.

Reply:
The script continues to prompt the user on invalid input

```
a@ubuntu:~$ msfconsole

 ** Welcome to Metasploit Framework Initial Setup **
    Please answer a few questions to get started.


Would you like to use and setup a new database (recommended)? testing
Please answer yes or no.
Would you like to use and setup a new database (recommended)? foo bar baz
Please answer yes or no.
Would you like to use and setup a new database (recommended)? invalid_123
Please answer yes or no.
Would you like to use and setup a new database (recommended)? y
```